### ISSUE TITLE
missing method FormatDeadLetterPath

### ISSUE THREAD

**Issue by hula100** [Azure/azure-sdk-for-python#547]
In SDK for .NET, there is a mehod "FormatDeadLetterPath" to get dead-letter queue path, but seems missing here.

**Comment by lmazuel** [comment]
You are right, we didn't have these helper functions. 
Thank you for your report, we will try to prioritize this soon.

In the meantime, since these functions are just string building, if you want to do a PR I will be happy to check it :-)

**Comment by lmazuel** [comment]
I released yesterday azure-servicebus 0.20.3 with these methods. They don't have exactly the same name, since the Python client has not the same interface, but the purpose is the same:
```python
dead_letter_name = ServiceBusService.format_dead_letter_subscription_name(
    'MySubscription')
dead_letter_name = ServiceBusService.format_dead_letter_queue_name(
    'MyQueue')
```

Thank you for the feedback!
